Alexandria/boardgame-backend
# Feature/typescript eslint

## Reference

[34](https://crema.atlassian.net/browse/BA-34)

## Overview
 I added a linter to my project. I also installed prettier.
I updated my README to include instructions on how to undo all migrations. 

### Acceptance Criteria

- [x] Successful run/compile after it’s installed in indication of success

### Demo
![Quick demo of the typescript-eslint linter](https://cl.ly/c9a1c2664d1f/Screen%252520Recording%2525202019-07-01%252520at%25252009.58%252520AM.gif)


## Reviewers' Checklist (go ahead, check 'em)


## Testing Instructions
Before testing this build you will need to have postgres installed and have a database already created.
Please look at the readme to see how to set up your environment variables.

1. Checkout branch feature/typescript-eslint
2. If you have tables built in your database run `npx sequelize db:migrate:undo:all`, if not skip this step
2. Run `npx sequelize db:migrate`
3. Run `npx sequelize db:seed:all`
